recipes/python-lambda-4dn/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "python-lambda-4dn" %}
+{% set version = "0.12.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 41765ea05224c035e146c89d60ef23d0d2c043ccc79e380cbeae60143547b1dc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
```

Please add
```suggestion
requirements:
  build:
    - pip
```

The build errors with "no module named pip".